blocks/library/embed/index.js

```diff
@@ -22,6 +26,23 @@ registerBlock( 'core/embed', {
 	edit( { attributes, setAttributes, focus, setFocus } ) {
 		const { url, title, caption } = attributes;
 
+		if ( ! url ) {
+			return (
+				<div className="blocks-embed is-placeholder">
```

Comment:
Regarding this question and the above about splitting out into a separate method... I'm not sure we can know the answer until we implement the placeholders for a couple of other blocks.  I vote we merge something like this structure and iterate later.